feat(nestjs): Add SentryTraced feature (#10610)
getsentry/sentry-docs: docs/platforms/javascript/guides/nestjs/features/sentry-traced-decorator.mdx

---
title: SentryTraced Decorator
description: "Learn about Sentry's SentryTraced decorator."
---

<Note>
  The `@SentryTraced` decorator is available from version `@sentry/nestjs` 8.15.0 and up.
</Note>

> The NestJS SDK now includes a @SentryTraced decorator that allows you to collect spans for any function it decorates.

Reviewer: ```suggestion
The NestJS SDK includes a `@SentryTraced` decorator that will create spans whenever the method it decorates is invoked.
```

> The NestJS SDK now includes a @SentryTraced decorator that allows you to collect spans for any function it decorates.

Reviewer 2: ```suggestion
The NestJS SDK now includes a `@SentryTraced` decorator that allows you to collect spans for any function it decorates.
```

This feature enables more granular performance monitoring and helps you gather detailed insights into the performance
of individual functions within your application.

> To get started import `@SentryTraced` from `@sentry/nestjs` and use it to decorate a function of your choice.

Reviewer: ```suggestion
To get started import `SentryTraced` from `@sentry/nestjs` and use it to decorate a function of your choice:
```

> ```jsx

Reviewer: ```suggestion
```typescript
```
